### PR TITLE
Add LOH output file headers with description to README

### PR DESCRIPTION
**Pull request by sakshamphul**
## Description

This PR adds a section in README to describe output file headers

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have committed any related changes to the PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,57 @@
 # D3B:Loss of Heterozygosity (LOH)
 
-LOH CWL workflow assesses the loss of heterozygosity(LOH) in the tumor for rare germline variants (gnomad_3_1_1_AF_popmax < 0.01).
+This CWL workflow assesses the loss of heterozygosity(LOH) in the tumor for rare germline variants (gnomad_3_1_1_AF_popmax < 0.01). This workflow has been deployed on [cavatica](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/apps/Loss_of_Heterozygosity) with a [proband only run](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/tasks/522d128a-2195-4c9c-8339-1709da16821d/) and a [trio run](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/tasks/d7f6b667-35ef-46a7-a666-970a78ef3175/)
```
Comment (dmiller15) [inline README.md:3]: So to everyone who isn't a member of this project, these links are pretty useless.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,57 @@
 # D3B:Loss of Heterozygosity (LOH)
```
Comment (dmiller15) [inline README.md:1]:
```suggestion
# D3b: Loss of Heterozygosity (LOH)
```

File: README.md (hunk 1 above)
Comment (sakshamphul) [reply README.md:3]: @dmiller15 I see, so what's the best solution to this ?

Comment (dmiller15) [reply README.md:3]: A public app?

Comment (sakshamphul) [reply README.md:3]: A ticket is up for a public app, meanwhile can I go forward without changing these lines?

Comment (dmiller15) [reply README.md:3]: Wouldn't you want to hold off merging this until you have the public app link so you can point people to that?

**Review by sickler-alex — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sakshamphul) [reply README.md:3]: I am thinking to make a new PR for public release

File: README.md [hunk 3]
```diff
@@ -1,34 +1,57 @@
...
 ### Running it locally on a laptop?
 
 
-It is recommended to run this workflow on a system with a high number of CPUs and memory (>=16 GB). The basic requirement is a running Docker engine and CWL tools. Here is the command line to run the LOH workflow locally:
-
+It is recommended to run this workflow on a system with a high number of CPUs and memory (>=16 GB). The basic requirement is a running docker engine and CWL tools. Command line to run the LOH workflow locally is:
 
 ```
 cwltool workflow/run_LOH_app.cwl sample_input.yml
 ```
```
Comment (dmiller15) [inline README.md:27]: Not sure if this is meaningfully informative. `sample_input.yml` is very much a hand wave.

File: README.md (hunk 1 above)
Comment (dmiller15) [reply README.md:3]: Ok.

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**